appveyor.yml

```diff
@@ -14,9 +15,9 @@ cache:
 
 install:
   - SET PATH=C:\%ruby_version%\bin;%PATH%
-  - gem uninstall bundler -a -x -I
-  - gem update --system 2.7.9
-  - # gem install bundler -v '1.17.3'
+  - gem uninstall bundler -x
+  - # gem update --system 2.7.10
+  - gem install bundler -v 1.17.3
```

Comment:
can you summarize what these changes address?

Reply:
in order to support ruby 2.1/2.2, we must ensure that bundler version stays below 2 and 1.17.3 is the latest version under 2.

in case bundler > 2 gets installed as part of rubygems update (which has been the case for rubygems 3+), we need to remove all installed bundler versions > 2 and then install the latest version < 2 explicitly